package.json

```diff
@@ -1,15 +1,20 @@
 {
   "name": "hi-base64",
-  "version": "0.1.3",
+  "version": "0.2.0",
   "description": "A simple Base64 encode / decode function for JavaScript supports UTF-8 encoding.",
   "main": "src/base64.js",
   "devDependencies": {
-    "expect.js": "~0.3.1",
-    "jscoverage": "~0.5.9"
+    "coveralls": "^2.11.4",
+    "expect.js": "^0.3.1",
+    "jscoverage": "^0.6.0",
+    "mocha": "^2.3.4",
+    "mocha-lcov-reporter": "^1.0.0",
+    "uglifyjs": "^2.4.10"
```

Comment:
This will mean that a developer can't see what's the coverage to help further. By the way, this is your library :)